src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+
+        results.model_result.measure_result.front_scan = Bunch()
+        results.model_result.measure_result.front_scan = get_stats(
+            np.array(self.front_predictions, dtype='float32'))
+        results.model_result.measure_result.back_scan = Bunch()
+        results.model_result.measure_result.back_scan = get_stats(
+            np.array(self.back_predictions, dtype='float32'))
+        results.model_result.measure_result.threesixty_scan = Bunch()
+        results.model_result.measure_result.threesixty_scan = get_stats(
+            np.array(self.threesixty_predictions, dtype='float32'))
+
+        # TODO get bunch object of artifact results for each type of scan
+
+        results.model_result.artifact_results = Bunch()
+        results.model_result.artifact_results.front_scan = get_artifact_result(
+            self.artifact_front, self.front_predictions)
+        results.model_result.artifact_results.back_scan = get_artifact_result(
+            self.artifact_back, self.back_predictions)
+        results.model_result.artifact_results.threesixty_scan = get_artifact_result(
+            self.artifact_threesixty, self.threesixty_predictions)
+
+        results_json_string = json.dumps(results)
+        self.results_json_object = json.loads(results_json_string)
+
+    def update_measure_table_and_blob(self, model_id, destination_folder):
+        '''
+        Update the result to measure_result table
+        '''
+        filename, folder = get_file_name(
+            destination_folder, model_id, self.qr_code[0], 'test')
+
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+
+        try:
+            with open(r'{0}{1}'.format(folder, filename), 'w') as json_file:
+                json.dump(self.results_json_object, json_file, indent=2)
+        except Exception as error:
+            print(error)
+
+        result_file = '{0}{1}'.format(folder, filename)
+
+        # TODO update results to database
+        flag = rgutils.upload_result(
+            result_file,
+            self.measure_id[0],
+            self.main_connector)
+
+        if flag == 0:
+            print("could not update to database")
+            return False
+
+        return True
+
+        # TODO send results to storage queue
+
+        # flag = rgutils.upload_to_queue(storage_account_name, result_file, main_connector)
+
+        # if flag != 0:
+        #    move_processed_data(result_file, 'measure_result', 'processed')
+        #    print(id[0])
+        # print(scan_measure_id[0][0])
+
+
+def main():
+    if len(sys.argv) != 3:
+        print("Please provide model_id and endpoint name.")
+        exit(1)
+
+    # destination_folder = str(sys.argv[1])
+    # db_connection_file = str(sys.argv[2])
+    # storage_account_name = str(sys.argv[3])
+    model_id = str(sys.argv[1])
+    service = str(sys.argv[2])
+    # calibration_file = str(sys.argv[6])
+    # container_name = str(sys.argv[7])
```

Comment:
I am not a big fan of using `sys.argv` directly. 

I see the usage is:
`/usr/bin/python3 ~/src/result_gen.py 20190723-1119_2550-638weight aci-pointnet-weight-20190723-ci`

I would be more readible if we name the parameters, e.g.:
`/usr/bin/python3 ~/src/result_gen.py --model-id 20190723-1119_2550-638weight --service aci-pointnet-weight-20190723-ci`